aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -129,24 +147,42 @@ module aptos_framework::staking_config {
         staking_config.rewards_rate_denominator = new_rewards_rate_denominator;
     }
 
+    /// Update the joining limit %.
+    /// Can only be called as part of the Aptos governance proposal process established by the AptosGovernance module.
+    public fun update_voting_power_increase_limit(
+        aptos_framework: &signer,
+        new_voting_power_increase_limit: u64,
+    ) acquires StakingConfig {
+        system_addresses::assert_aptos_framework(aptos_framework);
+        assert!(
+            new_voting_power_increase_limit > 0 && new_voting_power_increase_limit <= 100,
```

Comment:
need to update to 50 too, maybe we need to have a function for assertion or added a test for value > 50

Reply:
We already have a test for it. Updated both to respect the 50% limit.